Assets/BossRoom/Scripts/Shared/Net/ConnectionManagement/ClientGameNetPortal.cs

```diff
@@ -149,7 +162,7 @@ private void OnDisconnectOrTimeout(ulong clientID)
         /// <param name="portal"> </param>
         /// <param name="ipaddress">the IP address of the host to connect to. (currently IPV4 only)</param>
         /// <param name="port">The port of the host to connect to. </param>
-        public static void StartClient(GameNetPortal portal, string ipaddress, int port)
+        public void StartClient(GameNetPortal portal, string ipaddress, int port)
```

Comment:
Since this is no longer static, could we remove the portal parameter and use the m_Portal field instead? Same thing for other similar methods in this class